portal-api/src/crypto/utils.js

```diff
@@ -79,7 +76,46 @@ function issueJWT(user) {
   };
 }
 
+/**
+ * @param {Object} user We need this to set the JWT `sub` payload property to the MongoDB user ID
+ * @returns {Object} Token, expires in and session ID
+ */
+function issueValidUsernameAndPasswordJWT(user) {
+  return issueJWTWithExpiryAndPayload({
+    user,
+    expiresIn: '30m',
```

Comment:
I took this from the copy on the page fwiw

Reply:
The sign in link has a 30 minute expiry but that doesn't mean our token should have a 30 minute expiry time because you could e.g.:
- put in your username and password (which gets you a token)
- wait a couple minutes
- request a second sign in link

The 2nd sign in link is supposed to have a 30min expiry time but you'll actually only have 25mins left to use it because the token only has 25 mins left before it expires

Reply:
I'll raise a separate sub-task for this